components/net/tests/fetch.rs

```diff
@@ -138,6 +138,7 @@ fn test_fetch_blob() {
         fn process_response(&mut self, _: &Response) {}
         fn process_response_chunk(&mut self, chunk: Vec<u8>) {
             self.buffer.extend_from_slice(chunk.as_slice());
+            self.stream_chunk(buffer.as_slice());
```

Comment:
This is a unit test for unrelated code; we should be modifying components/script/fetch.rs instead.

Reply:
Oops, had both files open and made changes in the wrong one. Fixed it now